.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules
 .vim
-.DS_Store
+.DS_Store
```

Comment:
Can you add a newline to the end of this file?

Reply:
Yep, done